explorer/frontend/public/accounts/known-accounts.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Should this file be a config file provide at build time?  So that it can be different per chain(mainnet/testnet/etc)

Reply:
Additionally I made a config replaceable via docker volume as well as account images. So there is no need to rebuild.